src/components/UserSelector.tsx

```diff
@@ -17,9 +20,12 @@ export const UserSelector: React.FC<Props> = ({
   // `users` are loaded from the API, so for the performance reasons
   // we load them once in the `UsersContext` when the `App` is opened
   // and now we can easily reuse the `UserSelector` in any form
-  const users = useContext(UserContext);
+  // const users = useContext(UserContext);
   const [expanded, setExpanded] = useState(false);
 
+  const { value: users, isLoading, error } = useAppSelector((state: { users: { value: User[], isLoading: boolean, error: any } }) => state.users);
```

Comment:
useAppSelector should have defined all those types, you shouldn't doing it when passing the data